package.json

```diff
@@ -16,8 +16,8 @@
     "start": "nodemon ./src/server.js --exec 'npm run lint && node --inspect' | bunyan",
     "start:dev": "export NODE_ENV=development && yarn start",
     "test": "NODE_ENV=test nodemon ./src/test.js",
-    "test:ci": "NODE_ENV=test nyc --reporter=lcov --reporter=html node ./src/test.js --ci",
-    "test:gitlab": "NODE_ENV=testgitlab nyc --reporter=lcov --reporter=html node ./src/test.js --ci",
+    "test:ci": "NODE_ENV=test nyc --reporter=lcov node ./src/test.js --ci",
```

Comment:
@jrassa I'd like to add the html reporter back in to this. It helps to visualize the coverage on a local dev environment without requiring any other infrastructure. Any objections? Thoughts?

Reply:
Sorry, I meant to explain that change.  The lcov report also generates the html report.  Including both causes it to generate 2 copies of the html report.

https://istanbul.js.org/docs/advanced/alternative-reporters/#lcov

Reply:
That's amazing! I've been overlooking that for quite a while. Thanks again @jrassa !